tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md

```diff
@@ -0,0 +1,113 @@
+
+Configure the pallets by implementing their `Config` trait and update the runtime macro to include the new pallets:
+
+1. Add the `OriginCaller` import:
+    ```rust
+    --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/config-mod.rs:64:64'
+    ```
+
+2. Implement the [`Config`](https://paritytech.github.io/polkadot-sdk/master/pallet_utility/pallet/trait.Config.html){target=\_blank} trait for both pallets at the end of the `runtime/src/config/mod.rs` file:
+
+    ```rust
+    --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/config-mod.rs:313'
+    ```
+
+3. Locate the `#[frame_support::runtime]` macro in the `runtime/src/lib.rs` file and add the pallets:
+
+    ```rust hl_lines="5-9"
+    --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/lib.rs:250:252'
+    ...
+    --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/lib.rs:312:317'
+    ```
+
+## Recompile the Runtime
+
+After adding and configuring your pallets in the runtime, the next step is to ensure everything is set up correctly. To do this, recompile the runtime using the following command:
+
+```bash
+cargo build --release
+```
+
+This command ensures the runtime compiles without errors, validates the pallet configurations, and prepares the build for subsequent testing or deployment.
+
+## Run your Chain Locally
+
+Launch your parachain locally and start producing blocks:
+
+1. Create a new chain specification file with the updated runtime:
+
+    ```bash
+    chain-spec-builder create -t development \
+    --relay-chain paseo \
+    --para-id 1000 \
+    --runtime ./target/release/wbuild/parachain-template-runtime/parachain_template_runtime.compact.compressed.wasm \
+    named-preset development
+    ```
+
+2. Start the omni node with the generated chain specification:
+
+    ```bash
+    polkadot-omni-node --chain ./chain_spec.json --dev
+    ```
+
+3. Verify you can interact with the new pallets using the [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/extrinsics){target=\_blank} interface. Navigate to the **Extrinsics** tab and check that you can see both pallets:
+    - Utility pallet
+
+        ![](/images/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/add-pallets-to-runtime-1.webp)
+    
+
+    - Custom pallet
+
+        ![](/images/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/add-pallets-to-runtime-2.webp)
+
+## Where to Go Next
+
+<div class="grid cards" markdown>
+
+-   <span class="badge tutorial">Tutorial</span> __Deploy on Paseo TestNet__
+
+    ---
+
+    This guide walks you through the journey of deploying your Polkadot SDK blockchain on Paseo, detailing each step to a successful TestNet deployment.
+
+    [:octicons-arrow-right-24: Execute a Runtime Upgrade](/tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade/)
```

Comment:
This link and the link text are for a runtime upgrade but the title says Deploy on Paseo TestNet.

I don't see the Execute a Runtime Upgrade in the left nav but I can see the page exists if I type in the URL myself. Please fix this card to match where it should go and update the navigation to add the runtime upgrade page.